build.sh

```diff
@@ -3,7 +3,27 @@
 #################################################################
 # Override netlify.toml with centrally-hosted netlify.toml file #
 #################################################################
-curl https://raw.githubusercontent.com/opengovsg/isomer-build/master/netlify.toml -o /opt/build/repo/netlify.toml
+curl https://raw.githubusercontent.com/opengovsg/isomer-build/master/overrides/netlify.toml -o /opt/build/repo/netlify.toml
+
+###################################################################
+# Obtain config override file to enforce plugins and remote theme #
+###################################################################
+curl https://raw.githubusercontent.com/opengovsg/isomer-build/master/overrides/_config-override.yml
+
+#####################################################
+# Delete custom plugins from _plugins folder if any #
+#####################################################
+rm -rf _plugins
+
+#################################################
+# Check that Gemfile has not been tampered with #
+#################################################
+curl https://raw.githubusercontent.com/opengovsg/isomer-build/master/overrides/Gemfile -o Gemfile-template
+diff_line_count_gemfile =$(diff Gemfile Gemfile-template | wc -l)
+if (( diff_line_count_gemfile > 0 )); then
```

Comment:
typo, should be `$diff_line_count_gemfile`

Reply:
As discussed, we verified that this line of code works

Reply:
Some small issues in https://app.netlify.com/sites/a-test-v4-staging/deploys/610271407bf6ec000763fd93, see lines `133, 134`

Reply:
The issue was a typo, fixed in c676a74